amanuensis/blueprints/admin.py

```diff
@@ -255,13 +268,13 @@ def update_associated_user_role():
         raise UserError("A user_id and or an associated_user_email is required for this endpoint.")
 
     project_id = request.get_json().get("project_id", None)
-    role = request.get_json().get("role", None)
-    if request.method == "PUT":     
-        if not role:
-            raise UserError("A role is required for this endpoint")
-        if role not in admin.get_codes_for_roles():
-            raise NotFound("The role {} is not in the allowed list, reach out to pcdc_help@lists.uchicago.edu".format(role))
-    
+    if not project_id:
```

Comment:
The project_id here is just a number coming from the payload could you do a GET from the DB to check if the project exists? I think calling this 
https://github.com/chicagopcdc/amanuensis/blob/81cb4f3f863b5e5e35c559f07f03bbd8834e8721/amanuensis/resources/project/__init__.py#L73